frontend/SearchPane.js

```diff
@@ -53,14 +53,6 @@ class SearchPane extends React.Component {
   }
 
   onDocumentKeyDown(e) {
-    if (e.keyCode === 191) { // forward slash
-      var doc = ReactDOM.findDOMNode(this).ownerDocument;
-      if (!this.input || doc.activeElement === this.input) {
-        return;
-      }
```

Comment:
@Timothee Thanks! Just change to check `doc.activeElement.nodeName === 'INPUT'` works good.